.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+<!--
+# Have you read capa's CONTRIBUTING guide?
+It contains helpful information about how to contribute to capa. Check https://github.com/fireeye/capa/blob/master/.github/CONTRIBUTING.md
+
+based on https://embeddedartistry.com/blog/2017/08/04/a-github-pull-request-template-for-your-projects/
+-->
+
+### Description
+
+closes # (issue)
+
+<!-- Description of the PR -->
```

Comment:
What about adding some more detail:

`<<!-- Describe the changes in this PR. Include relevant motivation and context, as it helps reviewing your changes. -->`

I would also add the `closes # (issue)` after that line

Reply:
See c0c9ea3403c449ce7f4c9befd1065b59f1ac2203